### PR TITLE
[README] Add package version in all README

### PR DESCRIPTION
**Pull request by EtienneM**

### CODE REVIEW

File: gomock_generator/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# GoMock Generator
+# GoMock Generator v1.2.1
```
Comment (brandon-welsch) [inline gomock_generator/README.md:1]: Can you modify to have the same pattern as the others?

==> Package `gomock_generator` v1.2.1

File: graceful/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-## Graceful
+## Package `graceful` v1.0.1
```
Comment (brandon-welsch) [inline graceful/README.md:1]: The latest version of `graceful` package is v1.0.0 :)

image

File: httpclient/README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+# Package `httpclient` v1.0.1
```
Comment (brandon-welsch) [inline httpclient/README.md:1]: Same as graceful, latest version is v1.0.0

File: io/README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Packge `io`
+# Packge `io` v1.0.0
```
Comment (brandon-welsch) [inline io/README.md:1]: Small typo here

File: logger/README.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Logger
+# Package `logger` v1.0.0
```
Comment (brandon-welsch) [inline logger/README.md:1]: There is a `logger/v7.1.0` tag, should we keep it or remove it ?

File: retry/README.md [hunk 6]
```diff
@@ -1,4 +1,4 @@
-# Retryer
+# Package `retryer` v1.0.0
```
Comment (brandon-welsch) [inline retry/README.md:1]: `retry` not `retryer`

**Review by brandon-welsch — CHANGES_REQUESTED** [review]

File: logger/README.md (hunk 5 above)
Comment (EtienneM) [reply logger/README.md:1]: I think the tag can safely be removed.

Comment (brandon-welsch) [reply logger/README.md:1]: Okay, I'll delete it :)

**Review by brandon-welsch — APPROVED** [review]
LGTM

**Pull request merged**